backend/clubs/models.py

```diff
@@ -698,6 +698,31 @@ def send_approval_email(self, request=None, change=False):
                 reply_to=settings.OSA_EMAILS + [settings.BRANDING_SITE_EMAIL],
             )
 
+    def get_latest_and_latest_approved_versions(self):
+        """
+        Returns the latest and latest approved versions of club key fields
+        [name, description, image] contents that trigger reapproval.
+        """
+        latest_approved_version = (
+            self.history.filter(approved=True).order_by("-history_date").first()
+        )
+        latest_version = self.history.order_by("-history_date").first()
+
+        if (
+            not latest_version
+            or not latest_approved_version
+            or (latest_approved_version == latest_version)
+        ):
+            return {}
+
+        diff = latest_version.diff_against(latest_approved_version)
```

Comment:
If we're only reporting name, description, and image changes, do we need to diff every field? [Hint](https://django-simple-history.readthedocs.io/en/latest/history_diffing.html#history-diffing)

Reply:
Yep, in retrospect I'm not sure if this line is strictly necessary in the first place if we're computing diffs on the frontend (as discussed) anyways. We could compute on the backend but it would require more detailed serialization and not necessarily be much more performant at this scale.

Reply:
Yeah, agreed. I'm sure React has some great diff libraries. I think it'd be cleaner to expose something that fetches the most recently approved version of a club, and we can do diffing on the frontend. 

I'm also not sure where the `pending_clubs` endpoint would be used. We already know which clubs we want to diff (e.g. the ones in the approval queue), and I can't see where we'd use this outside of the queue. 

@Porcupine1 it'd be helpful to iron out the details with Julian further.

Reply:
I remember we discussed that initially we were just the calling the clubs viewset list function with the approved = false and active = true query/search params. I added `pending_clubs` because we are now returning pending clubs data